common/main/java/com/couchbase/lite/AbstractQuery.java

```diff
@@ -65,7 +65,7 @@ public void put(@NonNull ListenerToken token, @NonNull C4QueryObserver observer)
         public void start(@NonNull ChangeListenerToken<QueryChange> token) {
             synchronized (liveQueries) {
                 final C4QueryObserver observer = liveQueries.get(token);
-                if (observer != null) { observer.setEnabled(true); }
+                if (observer != null) { observer.enable(); }
```

Comment:
This doesn't seem be guard by database exclusive lock. I might have missed something.

Reply:
The caller seizes the db lock.  It is already held when this method is called

Reply:
I see.